system-variables.md

```diff
@@ -1717,6 +1725,13 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 范围：`[1, 100000]`
 - 这个变量用来控制 General Plan Cache 最多能够缓存的计划数量。
 
+### `tidb_gogc_tuner_threshold`
+
+- 作用域：GLOBAL
+- 是否持久化到集群：否
+- 默认值：`0.6`
+- 范围：`[0, 0.9)`
```

Comment:
```suggestion
- 范围：`[0, 18446744073709551615)`
```

Reply:
I think `0.9` is correct. It is the source code wrong. @hawkingrei , correct `MaxValue` of this variable.

Reply:
> I think `0.9` is correct. It is the source code wrong. @hawkingrei , correct `MaxValue` of this variable.

OK